Documentation/operations/upgrade.rst

```diff
@@ -336,6 +336,9 @@ Annotations:
 ------------------
 
 * Cilium has bumped the minimal Kubernetes version supported to v1.13.0.
+* When using the ENI based IPAM in conjunction with the ``--eni-tags`` failures
+  to create tags are treated as errors which will result in ENIs not being
+  created.
```

Comment:
I think it would be helpful if we provided a resolution to the user as well. Here's a suggestion, but feel free to word it how you see fit.

```suggestion
* When using the ENI-based IPAM in conjunction with the ``--eni-tags``, failures
  to create tags are treated as errors which will result in ENIs not being
  created. Ensure that the ``ec2:CreateTags`` IAM permissions are granted.
```

Reply:
Your wording sgtm!